x-pack/plugins/security_solution/public/management/common/constants.ts

```diff
@@ -11,7 +11,9 @@ import { ManagementStoreGlobalNamespace, AdministrationSubTab } from '../types';
 // --[ ROUTING ]---------------------------------------------------------------------------
 export const MANAGEMENT_ROUTING_ENDPOINTS_PATH = `${MANAGEMENT_PATH}/:tabName(${AdministrationSubTab.endpoints})`;
 export const MANAGEMENT_ROUTING_POLICIES_PATH = `${MANAGEMENT_PATH}/:tabName(${AdministrationSubTab.policies})`;
-export const MANAGEMENT_ROUTING_POLICY_DETAILS_PATH = `${MANAGEMENT_PATH}/:tabName(${AdministrationSubTab.policies})/:policyId`;
+export const MANAGEMENT_ROUTING_POLICY_DETAILS_FORM_PATH = `${MANAGEMENT_PATH}/:tabName(${AdministrationSubTab.policies})/:policyId/settings`;
+export const MANAGEMENT_ROUTING_POLICY_DETAILS_TRUSTED_APPS_PATH = `${MANAGEMENT_PATH}/:tabName(${AdministrationSubTab.policies})/:policyId/trustedApps`;
```

Comment:
(optional) Naming suggestion
Since we expect this tab to be renamed to `Artifacts` in the future (as we add more per-policy artifact support) maybe name the `const` `MANAGEMENT_ROUTING_POLICY_DETAILS_ARTIFACTS_PATH`?

Reply:
You mean the trustedApps tab? I thought this would remain with 'Trusted Apps' and then we will add a new tab as 'Event Filters'

Reply:
Adding more tabs will likely not scale, since there will be many and as we move forward with new artifacts, we will likely also have them support per-policy. Examples: Exceptions, Allow lists (IP), Deny lists (IP), etc.

Was also thinking that maybe the `const` name should match the URL route token - so maybe use the words `SETTINGS` and `ARTIFACTS`

Reply:
ok, it make sense, but it's strange that the tab is named 'Trusted Applications' and the url is `/artifacts`

Reply:
Yeah, I agree. It should be temporarily as I would imagine that Event Filters might not be far behind. I'm ok if you leave it as is for now. I don't think renaming it in the future will be much of a refactor, but we would likely have to introduce another redirect